org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/contentassist/XbaseProposalProvider.java

```diff
@@ -128,6 +128,155 @@ public boolean apply(IEObjectDescription input) {
 	@Inject
 	private SyntaxFilteredScopes syntaxFilteredScopes;
 	
+	protected class XbaseProposalCreator extends DefaultProposalCreator {
```

Comment:
maybe static inner class or not?

Reply:
parent is not static as well and it and the sub use stuff from outer context

Reply:
just wanted to make sure that this was a decision that way